### PR TITLE
Edit docs to fork

### PR DESCRIPTION
**Pull request by liortct**
Address #11

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -66,20 +66,20 @@ If you are proposing a feature:
 
 ## Setting Up the Code for Local Development
 
-Here's how to set up `cookiecutter` for local development.
+Here's how to set up `cookiejinja` for local development.
```
Comment (yuvaldolev) [inline CONTRIBUTING.md]:
```suggestion
Here's how to set up `cookieninja` for local development.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -66,20 +66,20 @@ If you are proposing a feature:
...
 
-1. Fork the `cookiecutter` repo on GitHub.
+1. Fork the `cookiejinja` repo on GitHub.
```
Comment (yuvaldolev) [inline CONTRIBUTING.md]:
```suggestion
1. Fork the `cookieninja` repo on GitHub.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,20 +66,20 @@ If you are proposing a feature:
...
 2. Clone your fork locally:
 
    ```bash
-   git clone git@github.com:your_name_here/cookiecutter.git
+   git clone git@github.com:your_name_here/cookiejinja.git
```
Comment (yuvaldolev) [inline CONTRIBUTING.md]:
```suggestion
   git clone git@github.com:your_name_here/cookieninja.git
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -317,6 +317,7 @@ New features require a +1 from 2 other core committers (besides yourself).
 ### Process: Roadmap
 
 The roadmap located [here](https://github.com/cookiecutter/cookiecutter/milestones?direction=desc&sort=due_date&state=open)
+TODO: change road map to cookieninja
```
Comment (yuvaldolev) [inline CONTRIBUTING.md]: Let's just update this to our milestones page

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -43,7 +43,7 @@ Whenever possible, it's best to try and implement feature ideas as separate proj
 
 ### Write Documentation
 
-Cookiecutter could always use more documentation, whether as part of the official Cookiecutter docs, in docstrings, or even on the web in blog posts, articles, and such.
+Cookieninja could always use more documentation, whether as part of the official Cookiejinja docs, in docstrings, or even on the web in blog posts, articles, and such.
```
Comment (yuvaldolev) [inline CONTRIBUTING.md]:
```suggestion
Cookieninja could always use more documentation, whether as part of the official Cookieninja docs, in docstrings, or even on the web in blog posts, articles, and such.
```

File: docs/advanced/template_extensions.rst [hunk 6]
```diff
@@ -51,9 +49,7 @@ Would output:
 Random string extension
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-*New in Cookiecutter 1.7*
-
-The ``cookiecutter.extensions.RandomStringExtension`` extension provides a ``random_ascii_string`` method in templates that generates a random fixed-length string, optionally with punctuation.
+The ``cookiejinja.extensions.RandomStringExtension`` extension provides a ``random_ascii_string`` method in templates that generates a random fixed-length string, optionally with punctuation.
```
Comment (yuvaldolev) [inline docs/advanced/template_extensions.rst]:
```suggestion
The ``cookieninja.extensions.RandomStringExtension`` extension provides a ``random_ascii_string`` method in templates that generates a random fixed-length string, optionally with punctuation.
```

**Review by yuvaldolev — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (liortct) [reply CONTRIBUTING.md]: solved

File: CONTRIBUTING.md (hunk 2 above)
Comment (liortct) [reply CONTRIBUTING.md]: solved

File: CONTRIBUTING.md (hunk 3 above)
Comment (liortct) [reply CONTRIBUTING.md]: solved

File: CONTRIBUTING.md (hunk 4 above)
Comment (liortct) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 5 above)
Comment (liortct) [reply CONTRIBUTING.md]: fixed

File: docs/advanced/template_extensions.rst (hunk 6 above)
Comment (liortct) [reply docs/advanced/template_extensions.rst]: solved

**Review by yuvaldolev — APPROVED** [review]
Nice! :)

**Pull request merged**